internal/cmd/keepassxctemplatefuncs.go

```diff
@@ -76,30 +78,57 @@ func (c *Config) keepassxcAttachmentTemplateFunc(entry, name string) string {
 		data := mustValue(os.ReadFile(tempFilename))
 		must(os.Remove(tempFilename))
 		return string(data)
+	case keepassxcModeBuiltin:
+		c.Keepassxc.cache[entry] = c.keepassxcBuiltinExtractValues(entry, keepassxcBuiltinMapAttachmentCache)
+		if data, ok := c.Keepassxc.cache[entry]; ok {
+			return data[name]
+		}
+		panic(fmt.Sprintf("attachment %s of entry %s not found", entry, name))
```

Comment:
The `%s` parameters are reversed. The message isn't *quite* right, because this is only checking that `entry` is present, not that `entry[name]` is present.

```suggestion
		panic(fmt.Sprintf("entry %s attachment %s not found", entry, name))
```